packages/server-wallet/src/chain-service/__chain-test__/chain-service.test.ts

```diff
@@ -360,7 +361,7 @@ describe('concludeAndWithdraw', () => {
     if (!transactionResponse) throw 'Expected transaction response';
     await transactionResponse.wait();
 
-    mineBlocks();
+    await mineBlocks();
```

Comment:
I could be wrong, but I don't think this `await` is needed since we are blocking on channel finalization promise `p` in the line below.